operations/image_management/Import_External_Image_to_IMS.md

```diff
@@ -84,45 +81,42 @@ the following procedure can be used to add this external image to IMS and upload
         ncn# cray artifacts create boot-images $IMS_IMAGE_ID/$IMS_ROOTFS_FILENAME $ROOTFS_FILENAME
         ```
 
-1.  Optionally, upload the kernel for the image to S3.
+1. Optionally, upload the kernel for the image to S3.
 
     > In this example, the relative path to the kernel from the working directory is
     > `image-root/boot/vmlinuz`.
 
     ```bash
-    ncn# export IMS_KERNEL_FILENAME=vmlinuz
-    ncn# cray artifacts create boot-images $IMS_IMAGE_ID/$IMS_KERNEL_FILENAME \
-            image-root/boot/$IMS_KERNEL_FILENAME
-    ncn# export IMS_KERNEL_MD5SUM=`md5sum image-root/boot/$IMS_KERNEL_FILENAME | awk '{ print $1 }'`
+    ncn# IMS_KERNEL_FILENAME=vmlinuz
+    ncn# cray artifacts create boot-images $IMS_IMAGE_ID/$IMS_KERNEL_FILENAME image-root/boot/$IMS_KERNEL_FILENAME
+    ncn# IMS_KERNEL_MD5SUM=`md5sum image-root/boot/$IMS_KERNEL_FILENAME | awk '{ print $1 }'`
     ```
 
-1. Optionally, upload the initrd for the image to S3.
+1. Optionally, upload the `initrd` for the image to S3.
 
-    > In this example, the relative path to the initrd file from working directory is
+    > In this example, the relative path to the `initrd` file from working directory is
     > `image-root/boot/initrd`.
 
     ```bash
-    ncn# export IMS_INITRD_FILENAME=initrd
-    ncn# cray artifacts create boot-images $IMS_IMAGE_ID/$IMS_INITRD_FILENAME \
-    image-root/boot/$IMS_INITRD_FILENAME
-    ncn# export IMS_INITRD_MD5SUM=`md5sum image-root/boot/$IMS_INITRD_FILENAME | awk '{ print $1 }'`
+    ncn# IMS_INITRD_FILENAME=initrd
```

Comment:
ncn# doesn't seem right. I don't think this will be something you run from a storage node(101-103)

Reply:
Addressing this with https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-4815